sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectName=RPE :: expressjs template
```

Comment:
are these replaced by the init script?

Reply:
Not, it is not. Hopefully, it's something we can improve later on as well.